### PR TITLE
New jennica

### PR DESCRIPTION
**Pull request by JennicaLeClerc**
Did some things....

### CODE REVIEW

File: src/API/UserAPI.js
Comment (LutherNa) [inline src/API/UserAPI.js]:
```suggestion
	return APIQuery.put("users/password", passwords, axiosConfig(JWT));
```

**Review by LutherNa — CHANGES_REQUESTED** [review]
request approved contingent on updating to main after the other recent merges.

**Review by LutherNa — APPROVED** [review]

**Review by nfrederick023 — APPROVED** [review]
Looks good!

**Pull request merged**
